sw/host/opentitanlib/src/util/openocd.rs

```diff
@@ -292,6 +292,60 @@ impl OpenOcdServer {
             }
         }
     }
+
+    // Convert a RISC-V register name into a name that can be used
+    // with read_register32.
+    fn riscv_reg_name(&self, reg: &RiscvReg) -> &'static str {
+        match reg {
+            RiscvReg::GprByName(gpr) => match gpr {
+                RiscvGpr::GP => "gp",
+                RiscvGpr::SP => "sp",
+            },
+            RiscvReg::CsrByName(gpr) => match gpr {
+                RiscvCsr::DPC => "dpc",
+                RiscvCsr::PMPCFG0 => "pmpcfg0",
+                RiscvCsr::PMPCFG1 => "pmpcfg1",
+                RiscvCsr::PMPCFG2 => "pmpcfg2",
+                RiscvCsr::PMPCFG3 => "pmpcfg3",
+                RiscvCsr::PMPADDR0 => "pmpaddr0",
+                RiscvCsr::PMPADDR15 => "pmpaddr15",
+            },
+        }
```

Comment:
I think we need a default case here, maybe just panic on no match?

Reply:
I have decided to not go this route: if we add a register to `RiscvCsr` or `RiscvGpr` then the match will not be exhaustive and not compile. I think this is preferable to panicking since we expect to only add registers that we can handle. Does that sound acceptable?